packages/react-composites/src/composites/ChatComposite/file-sharing/index.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+export * from './FileCard';
+export * from './FileCardGroup';
+export * from './FileUploadButton';
+export * from './FileUploadHandler';
+export * from './UploadedFile';
+
+/**
+ * @internal
+ */
+export const ACSFilesMetadataKey = 'acsfiles';
```

Comment:
In general unless we feel strongly about this lets not use the abbreviation "acs"

Reply:
if this is just within the scope of the chat composite then we can say something like

filesharing_metadata_key instead?

Reply:
Was hoping to name the key something that allows partners to quickly identify that it's an ACS specific key (or UI library specific. )
`filesharing_metadata_key` seems too generic?
Also introduces snake case, which is not a standard naming practice in JS world compared to snake case.